server/log_rpc_server.go

```diff
@@ -78,12 +78,6 @@ func (t *TrillianLogRPCServer) QueueLeaves(ctx context.Context, req *trillian.Qu
 	}
 	leaves := depointerify(req.Leaves)
 
-	// TODO(al): Hasher must be selected based on log config.
-	th, _ := merkle.Factory(merkle.RFC6962SHA256Type)
```

Comment:
Why isn't this necessary anymore? Do we expect the personality to hash, or is it happening in another layer already?